language/gpt-j/README.md

```diff
@@ -0,0 +1,79 @@
+# GPT-J Reference Implementation
+
+### Setup Instructions
+
+```bash
+WORK_DIR=$PWD
+# Create Environment (conda)
+conda create -n llm python=3.9 -y
+conda activate llm
+conda install mkl mkl-include -y
+conda install gperftools jemalloc==5.2.1 -c conda-forge -y
+
+# install pytorch
+# you can find other nightly version in https://download.pytorch.org/whl/nightly/
+pip install https://download.pytorch.org/whl/nightly/cpu-cxx11-abi/torch-2.0.0.dev20230228%2Bcpu.cxx11.abi-cp39-cp39-linux_x86_64.whl
```

Comment:
Can you please explain why this particular version is needed as 2.0 is now officially released?

Reply:
Thank you @badhri-intel So, `pip install torch` is enough right?

Reply:
Yes, that would work as well.

Reply:
Thank you @badhri-intel for confirming. The instructions are very clean and should be easy to run. We'll add then to CK as soon as training submissions are done.